servatrice/src/servatrice_database_interface.cpp

```diff
@@ -1091,3 +1091,23 @@ QList<ServerInfo_ChatMessage> Servatrice_DatabaseInterface::getMessageLogHistory
 
     return results;
 }
+
+bool Servatrice_DatabaseInterface::isUserADonator(const QString &user)
+{
+    if (!checkSql())
+        return false;
+    
+    QSqlQuery *query = prepareQuery("SELECT donator from {prefix}_users WHERE name = :user_name");
```

Comment:
This is likely the wrong place to do it. Isn't there already a SELECT on _users when a person joins the server? Add the bypass column to that select instead of doing an extra query. It will be much more efficient in terms of db load.